mmv1/templates/terraform/examples/compute_interconnect_basic_test.tf.erb

```diff
@@ -3,7 +3,7 @@ data "google_project" "project" {}
 resource "google_compute_interconnect" "<%= ctx[:primary_resource_id] %>" {
   name                 = "<%= ctx[:vars]['interconnect_name'] %>"
   customer_name        = "internal_customer" # Special customer only available for Google testing.
-  interconnect_type    = "IT_PRIVATE"        # Special type only available for Google testing.
+  interconnect_type    = "DEDICATED"        # Special type only available for Google testing.
```

Comment:
Is the comment `# Special type only available for Google testing.` still be correct for `DEDICATED `?